stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -5124,16 +5124,27 @@ extension ${Self} {
     }
   }
 
+  /// Gets the native Dictionary hidden inside this NSDictionary
```

Comment:
Guidelines say:

Describe what a function or method _does_ and what it _returns_, omitting null effects and Void returns:

```swift
/// Inserts `newHead` at the beginning of `self`.
mutating func prepend(_ newHead: Int)

/// Returns a `List` containing `head` followed by the elements
/// of `self`.
func prepending(_ head: Element) -> List

/// Removes and returns the first element of `self` if non-empty;
/// returns `nil` otherwise.
mutating func popFirst() -> Element?
```